spec/dummy/app/views/pages/index.html.erb

```diff
@@ -39,16 +22,50 @@
 
 <% cache @app_props_server_render do %>
   <% puts "=" * 80 %>
-  <% puts "server rendering react component" %>
+  <% puts "server rendering react components" %>
   <% puts "=" * 80 %>
   <!-- Default for prerender is true for the app, set in config/react_on_rails.rb -->
   <%= react_component("App", @app_props_server_render, trace: true) %>
+  <hr/>
+
+  <h1>Server Rendered/Cached React Component Without Redux</h1>
+  <code>
+    <%% cache @app_props_server_render do %><br/>
+    <%%    = react_component("HelloWorld", @app_props_server_render, trace: true) %><br/>
```

Comment:
We should make the component generator "function" name more explicit. Like `HelloWorldComponentGenerator` or `HelloWorldApp`. @alexfedoseev What do you think? I think we might be using the suffix `App` to mean these generators.